extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeUtils.java

```diff
@@ -10,7 +10,6 @@
  * #L%
  */
 
-import com.boozallen.aiops.security.authorization.policy.AiopsAttribute;
```

Comment:
Q: Was this replaced by a corresponding AissembleAttribute, or is is no longer needed?

Reply:
The import is no longer needed which is why it was removed.